### DISCUSSION TITLE
Guide on adding a device

### DISCUSSION THREAD

**Discussion by Newbie2Quick** [rytilahti/python-miio#1136] [Q&A]
Just installed Python-miio. but kinda lost on how to add a device. Any guidance on how to add a device would be greatly appreciated. I have the ip address and token for my roborock vacuum. 

I assume it is through the terminal. PS> i am using Homebridge.

**Comment by rytilahti** [comment]
Hi,

this is just a python library (and accompanying console tool) to allow commanding these devices, so there is nothing to "add" itself.

For roborock vacuums, you can either use `mirobo` or `miiocli vacuum` commands to control the device, e.g.:
```
mirobo --ip <addr> --token <token> status
```
or
```
miiocli vacuum --ip <addr> --token <token> status
```

Is that what you are looking for? :-)
